README.md

```diff
@@ -214,6 +214,17 @@ A number of options are available to allow you to customise the SDK:
 
   A string of the ID of the container element that the UI will mount to. This needs to be an empty element. The default ID is `onfido-mount`.
 
+- **`language {Object} optional`**
+  An object that includes the selected locale. If `language` is not present the locale will default to `en`.
+  In order to override the default translation of a string it is possible to use the option `customStrings` that accepts an object containing the key and the new value. The keys are available with `./locales/en.js`
+
+    ```javascript
+    language: {
+      locale: 'es', // required
```

Comment:
This isn't required though right? We default to English

Reply:
Correct. That's not required. It's an example of how the language object should look if they want to specify a locale that is not English.

Reply:
Interesting that you already done this, but I'm not sure if the feature should already be there (or maybe advertised) considering that no other language can be chosen.

Reply:
Yes, you are right. I was thinking that this PR and the Spanish translation one would be merged and released together but this PR can actually be merged independently, so I could just remove the changes in the `README.md` and `CHANGELOG.md` and include them in the next PR